oem.json

```diff
@@ -608,22 +689,37 @@
     },
     {
         "id": "com.duokan.phone.remotecontroller",
-        "description": "Mi Remote Controller (https://play.google.com/store/apps/details?id=com.duokan.phone.remotecontroller)\nControl your electric appliances with your phone using Mi Remote.\n",
+        "label": "Mi Remote Controller",
+        "description": "Control your electric appliances with your phone using Mi Remote.",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.duokan.phone.remotecontroller"
+        ],
         "removal": "delete"
     },
     {
         "id": "com.duokan.phone.remotecontroller.peel.plugin",
-        "description": "Peel Mi Remote (https://play.google.com/store/apps/details?id=com.duokan.phone.remotecontroller.peel.plugin)\nPeel Mi Remote is a TV guide extension for Xiaomi Mi Remote by \"Peel Smart Remote\".\n",
+        "label": "Peel Mi Remote",
+        "description": "Peel Mi Remote is a TV guide extension for Xiaomi Mi Remote by \"Peel Smart Remote\".",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.duokan.phone.remotecontroller.peel.plugin"
+        ],
         "removal": "delete"
     },
     {
         "id": "com.elephanttek.faceunlock",
-        "description": "Standard FaceUnlock functionality?\nUnlock your device by simply looking at the display.\nFace unlock is bad for security and privacy:\nhttps://www.ubergizmo.com/2017/03/galaxy-s8-facial-unlock-photograph/\nhttps://www.kaspersky.com/blog/face-unlock-insecurity/21618/\nhttps://www.freecodecamp.org/news/why-you-should-never-unlock-your-phone-with-your-face-79c07772a28/",
+        "label": "com.elephanttek.faceunlock",
+        "description": "Standard FaceUnlock functionality?\nUnlock your device by simply looking at the display.",
```

Comment:
I thought it's upto users'  to decide. The links are already given there to make their own opinion about it.

Reply:
> I thought it's upto users' to decide. The links are already given there to make their own opinion about it.

We aren't deciding anything, we're simply helping the users by informing them that these features are bad. This requires a focus because biometric lock/unlocks are indeed a major breach of privacy and security even in countries with strict freedom acts or laws. Links are intended for additional context. _We should expect that most users won't even read those link texts._